docs/content/concepts/sui-move-concepts/patterns/witness.mdx

```diff
@@ -51,5 +51,5 @@ module examples::peace_guardian {
 
 This pattern is used in these examples:
 
-    - Liquidity pool
-    - Regulated coin
+- Liquidity pool
```

Comment:
Note: I fixed a formatting issue here, but there's a larger issue related to these examples not being available in the new docs site -- we may want to point to some other example (like "Create a Coin").